code/android-core-library/src/main/java/com/adobe/marketing/mobile/internal/util/FileUtils.kt

```diff
@@ -21,7 +21,7 @@ import java.io.InputStream
 import java.io.InputStreamReader
 import java.util.zip.ZipInputStream
 
-object FileUtils {
+internal object FileUtils {
```

Comment:
This utility has usages outside of the `internal` package with my changes for `CacheService`. What do you think about moving this to public utils?

Reply:
@prudrabhat Let us review and make it public after your changes are in. I prefer exposing only `extractFromZip` unless other extensions use any of the other methods.

Reply:
Sure. 👍🏼  `readAsString()` and `extractFromZip` are the ones with usage in rule download.